core/src/main/java/org/hibernate/ogm/persister/impl/OgmEntityPersister.java

```diff
@@ -526,6 +543,47 @@ private TupleContext createTupleContext() {
 		);
 	}
 
+
+	public void createIndex(OgmIndexSpec indexSpec)
+	{
+		gridDialect.createIndex(indexSpec);
+	}
+
+	public List<OgmIndexSpec> getIndexSpec()
+	{
+		return gridDialect.getIndexSpec(tableName,indexAnnotations);
+	}
+
+	private Map<String, Annotation> getOgmIndexAnnotations(Class cls) {
```

Comment:
I actually looked at the `option` path too.
But sadly, I cant recall why I dismissed using that ..., I will try again 
and this time. I'll take some note, if I get stuck again.